dandiapi/api/models/version.py

```diff
@@ -86,10 +86,11 @@ def copy(cls, version):
     @classmethod
     def citation(cls, metadata):
         year = datetime.datetime.now().year
-        name = metadata['name']
+        name = metadata['name'].rstrip('.')
         url = metadata['url']
+        version = metadata['version']
         # If we can't find any contributors, use this citation format
-        citation = f'{name} ({year}). Online: {url}'
+        citation = f'{name} ({year}). (Version {version}) [Data set]. DANDI archive. {url}'
```

Comment:
just some context since I wondered why `Data set` and not `Dataset`? It seems that it is how Datacite renders it  as well even though uses `Dataset` to describe the entry (why oh why?!):
![image](https://user-images.githubusercontent.com/39889/120840467-4caa4600-c538-11eb-9525-e4a34a9446ef.png)

Reply:
oh why indeed . sometimes i just stop asking.